src/resolvers/donationResolver.ts

```diff
@@ -366,14 +370,20 @@ export class DonationResolver {
         networkId: chainId,
         symbol: token,
       });
-      if (!tokenInDb) throw new Error(errorMessages.TOKEN_NOT_FOUND);
-      const acceptsToken = await isTokenAcceptableForProject({
-        projectId,
-        tokenId: tokenInDb.id,
-      });
-      if (!acceptsToken) {
-        throw new Error(errorMessages.PROJECT_DOES_NOT_SUPPORT_THIS_TOKEN);
+      let isTokenEligibleForGivback = false;
+      if (!tokenInDb && !project.organization.supportCustomTokens) {
+        throw new Error(errorMessages.TOKEN_NOT_FOUND);
+      } else if (tokenInDb) {
+        const acceptsToken = await isTokenAcceptableForProject({
+          projectId,
+          tokenId: tokenInDb.id,
+        });
+        if (!acceptsToken && !project.organization.supportCustomTokens) {
+          throw new Error(errorMessages.PROJECT_DOES_NOT_SUPPORT_THIS_TOKEN);
```

Comment:
I think we shouldn't stop the process from saving the donation, because even if the project don't support givbacks for this donation with that token, the donation was still done, so we should keep a record of that.

So we should log the error and set isTokenElegibleForGivback = false.
Because what I am seeing from the tests and code, is there is no case in which it is false.

Reply:
Actually I think the case we are missing is. If it accepts custom tokens but its not in db. We should let this case pass and not throw error.

Reply:
Does this case make isTokenEligibleForGivback = false? Will Confirm tomorrow.